### ISSUE TITLE
Hue Items error with ha-bridge-5.0.0rc2

### ISSUE THREAD

**Issue by TheCellMC** [bwssytems/ha-bridge#768]
I am getting this error when I navigate to the "Hue Devices" tap. I also tried doing so after pressing the button on the hue bridge.

`Get Hue Items Error: undefined with status: Server Error - 500`

Here are the logs
```
0-23-2017 21:57:22.928 | WARN | HTTP response code was not an expected successful response of between 200 - 299, the code was: HTTP/1.1 404 Not Found | com.bwssystems.HABridge.plugins.http.HTTPHandler
0-23-2017 21:57:22.932 | ERROR |   | spark.webserver.MatcherFilter
```

**Comment by TheCellMC** [comment]
I fixed this by downgrading to the  Release Candidate 1 for 5.0.0.
I think that it has to do with [this](https://github.com/bwssytems/ha-bridge/commit/5231eac4b089e7f60d9169ea78a6911303ec69fb) commit.
I also can't control the temperature of my lights when I change the whole group. However, I can change it individually.

**Comment by bwssytems** [comment]
This will be fixed in 5.0.0 with the new http handler
